### ISSUE TITLE
IN Query not working RxDB SqlLite premium

### ISSUE THREAD

**Issue by meabed** [pubkey/rxdb#4278]
Hi,

In RxDB SqlLite premium the in query doesn't work: 

```
# this doesnt work
collection.find({selector: {_id: { $in: ['a1','b2'] } }})

# this works 
collection?.find({selector: {_id: 'a1'} }})

```

In PouchDB it works fine as well - it only error when switching to the premium plugins.

Thank you

**Comment by pubkey** [comment]
Please make a PR with a test for it to this file: https://github.com/pubkey/rxdb/blob/master/test/unit/rx-storage-query-correctness.test.ts
In the background, these tests run also on the premium storages.

**Comment by meabed** [comment]
It's the react native with quick SQLite.

What test file would you need to reproduce this?

**Comment by meabed** [comment]
@pubkey I have made full working app to demonstrate the bug.

The code is [here](https://github.com/meabed/rxdb-react-native-demo)

it has both pouch db and premium plugin configured and replicating dummy data.


As the screens below: When using IN Filter in pouchdb it works fine, and it doesn't work in the premium plugins.

<img width="470" alt="image" src="https://user-images.githubusercontent.com/45731/211193390-a5971387-56f2-43f0-8c51-297a371864d3.png">


<img width="464" alt="image" src="https://user-images.githubusercontent.com/45731/211193288-0258e998-19c0-40e2-a757-7e5444225e52.png">

<img width="460" alt="image" src="https://user-images.githubusercontent.com/45731/211193293-dfb63bf1-2c12-4cb7-adbe-69120dce9421.png">

The code is pretty simple to navigate thro: 

- https://github.com/meabed/rxdb-react-native-demo/blob/master/src/screens/pouchdb/components/product-list.tsx

https://github.com/meabed/rxdb-react-native-demo/blob/master/src/screens/premium/components/product-list.tsx

**Comment by pubkey** [comment]
I could reproduce and fix it.
The problem occured when more then one array item was passed to $in.

**Comment by meabed** [comment]
Will this be released in 13.7 ?

**Comment by meabed** [comment]
@pubkey could this be fixed in 13.7 as well?

**Comment by pubkey** [comment]
I released rxdb premium version `13.17.3`, please try it out.

**Comment by meabed** [comment]
thank you @pubkey I haven't seen the release yet in npm.

**Comment by meabed** [comment]
Hey @pubkey - not sure if you seeing this, but this is not released yet

**Comment by pubkey** [comment]
@meabed  I had to do a release manually which was more complex then I thought because of the whole token access and encryption.
There is no RxDB 13.17.3, only the rxdb-premium was released as 13.17.3.

![release-premium](https://user-images.githubusercontent.com/8926560/211815719-6a4b7b8c-c840-4998-910b-c94ea3037917.png)

**Comment by meabed** [comment]
Tested and it's working ✅ Thank you.
